code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -191,6 +191,12 @@
 /mob/living/simple_animal/drone/cogscarab/ratvar_act()
 	fully_heal(TRUE)
 
+/mob/living/simple_animal/drone/cogscarab/can_speak_in_language(datum/language/dt)
```

Comment:
something like that, just something to prevent having to add this boilerplate to several different mobs just because they can't speak X language, you instead put in X language that ```/mob/Y``` cant speak it.